src/js/view/controls/components/timeslider.ts

```diff
@@ -305,7 +305,7 @@ class TimeSlider extends Slider {
             timeTipPct = (timeTipWidth - tolerance) / (2 * 100 * widthPct);
         }
         const safePct: number = parseFloat(Math.min(1 - timeTipPct, Math.max(timeTipPct, pct)).toFixed(3)) * 100;
-        style(timeTip.el, { left: safePct + '%' });
+        style(timeTip.el, { transform: `translateX(${safePct}%)`});
```

Comment:
This could also be in pixels, but since we are already using percentages, decided to keep with that.